lib/restrictedsession/restricted_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"testing"
 	"time"
 
-	go_cmp "github.com/google/go-cmp/cmp"
+	gocmp "github.com/google/go-cmp/cmp"
```

Comment:
Just call it `cmp`? I don't think the alias is necessary.

```suggestion
	"github.com/google/go-cmp/cmp"
```

Reply:
This package already has a method called `cmp` Go doesn't allow you to have a collision. I can rename that function but sounds like the same deal, rename something, so `cmp` can be used in a different place.

Reply:
It's alright, let's keep the alias.